src/colmap/sfm/incremental_mapper.h

```diff
@@ -118,6 +118,16 @@ class IncrementalMapper {
     // If reconstruction is provided as input, fix the existing image poses.
     bool fix_existing_images = false;
 
+    // Whether to use prior camera positions
+    bool use_prior_position = false;
+
+    // Whether to use a robust loss on prior locations
+    bool use_robust_loss_on_prior_position = false;
+
+    // Threshold on the residual for the robust loss
+    // (chi2 for 3DOF at 95% = 7.815)
+    double prior_position_loss_scale = 7.815;
+
```

Comment:
@ahojnnes Do you see a cleaner way to pass these options from the CLI to the PosePriorBA? The set of options will likely grow in the future with additional sensors...

Reply:
The structure is indeed getting a little bit out of control. I don't think we want to block this PR here, as this was an issue already before. We can probably group them more meaningfully or reuse the underlying options structs. Not sure.